src/test/java/com/philips/research/spdxbuilder/persistence/blackduck/BlackDuckClientTest.java

```diff
@@ -165,13 +165,15 @@ void findsByName() throws Exception {
                                     .put("releaseComments", DESCRIPTION)
                                     .put("distribution", DISTRIBUTION)
                                     .put("license", new JSONObject().put("spdxId", LICENSE))
+                                    .put("createdAt", "2021-02-15T09:47:02.105Z")
                                     .put("_meta", new JSONObject()
                                             .put("href", URI.create("https://server/something/" + VERSION_ID))))).toString()));
 
             final var projectVersion = client.findProjectVersion(PROJECT_ID, VERSION).orElseThrow();
 
             assertThat(projectVersion.getId()).isEqualTo(VERSION_ID);
             assertThat(projectVersion.getName()).isEqualTo(VERSION);
+            assertThat(projectVersion.getCreatedAt()).contains("2021-02-15T09:47:02.105Z");
```

Comment:
this will also change if we make this param typed as suggested in the earlier review as well.